pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -105,7 +110,7 @@ def serve_requests(self, num_requests=None):
                 event_message = self._resolve_event(event_message_length)
 
                 # instantiate event message
-                event = nuclio_sdk.Event.from_msgpack(event_message)
+                event = self._event_deserializer.deserialize(event_message)
```

Comment:
```suggestion
                deserializer = nuclio_sdk.EventDeserializerFactory.create(deserializer_kind)
                event = nuclio_sdk.Event.deserialize(event_message, decode_strings=self._decode_events)
```

Reply:
behavior parameter will lead to needlessly `if else`  kind of statements which will decrease performance.
since `self._decode_events` won't change during function lifetime, it creates noise per each event as oppose to create the deserializer once and reuse it many times.